<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="Hichem" name="StandAlone.ARender_Full_Fonction" tests="8" failures="1" timestamp="2020-11-09T10:58:52 WAT" time="7.090" errors="0">
  <testcase name="Brightness_Test" time="2.620" classname="StandAlone.ARender_Full_Fonction"/>
  <system-out/>
  <testcase name="Navigation_Button_Test" time="2.496" classname="StandAlone.ARender_Full_Fonction"/>
  <system-out/>
  <testcase name="Fullscreen_Test" time="1.318" classname="StandAlone.ARender_Full_Fonction"/>
  <system-out/>
  <testcase name="logo_Test" time="0.656" classname="StandAlone.ARender_Full_Fonction">
    <failure type="java.lang.AssertionError" message="ERROR ACCURRED : ARENDER VERSION IS NOT TRUE expected [Version 4.1.1-rc1 - 201020_144655] but found [Version 4.1.1-rc3 - 201028_110038]">
      <![CDATA[java.lang.AssertionError: ERROR ACCURRED : ARENDER VERSION IS NOT TRUE expected [Version 4.1.1-rc1 - 201020_144655] but found [Version 4.1.1-rc3 - 201028_110038]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at StandAlone.ARender_Full_Fonction.logo_Test(ARender_Full_Fonction.java:148)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- logo_Test -->
  <system-out/>
  <testcase name="Contrast_Test" classname="StandAlone.ARender_Full_Fonction">
    <skipped/>
  </testcase> <!-- Contrast_Test -->
  <system-out/>
  <testcase name="Check_Print_allFiles_fonctions" classname="StandAlone.ARender_Full_Fonction">
    <skipped/>
  </testcase> <!-- Check_Print_allFiles_fonctions -->
  <system-out/>
  <testcase name="Check_Print_WithAnnotation_fonctions" classname="StandAlone.ARender_Full_Fonction">
    <skipped/>
  </testcase> <!-- Check_Print_WithAnnotation_fonctions -->
  <system-out/>
  <testcase name="Check_Print_WithoutAnnotation_fonctions" classname="StandAlone.ARender_Full_Fonction">
    <skipped/>
  </testcase> <!-- Check_Print_WithoutAnnotation_fonctions -->
  <system-out/>
</testsuite> <!-- StandAlone.ARender_Full_Fonction -->
